src/Controller/TideApiController.php

```diff
@@ -343,29 +343,29 @@ private function resolveAliasPath(Request $request, $path, $site, $cid, array &$
 
     // Dispatch a GET_ROUTE event so that other modules can modify it.
     if ($code != Response::HTTP_BAD_REQUEST) {
-      $event_entity = NULL;
-      if ($entity) {
-        $event_entity = clone $entity;
-      }
-      $event = new GetRouteEvent(clone $request, $json_response, $event_entity, $code);
+      $event = new GetRouteEvent(clone $request, $json_response, (!empty($entity) ? clone $entity : NULL), $code);
```

Comment:
Is this required? Doesn't the old code achieve the same thing?

Reply:
`$event_entity` is not used, hence it's not necessary to define a variable.